example/k8s.yaml

```diff
@@ -0,0 +1,166 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: oc-agent-conf
+  labels:
+    app: opencensus
+    component: oc-agent-conf
+data:
+  oc-agent-config: |
+#    receivers:
+#      jaeger: {}
```

Comment:
I'm not sure if the `{}` works for agent config.

Reply:
done